packages/cli/src/commands/deploy/args.ts

```diff
@@ -21,6 +21,7 @@ export const help = () => `
       link                             Link local directory to a Vercel Project
       login                [email]     Logs into your account or creates a new one
       logout                           Logs out of your account
+      pull                 [filename]  Pull the Environment Variables and settings for your current Project
```

Comment:
```suggestion
      pull                 [filename]  Pull your Project Settings from the cloud
```

I think env vars are included in Project Settings 

![image](https://user-images.githubusercontent.com/229881/137534512-22eed67c-e8c4-405a-8c4d-b7ccae58f198.png)